lib/puppet/face/ca.rb

```diff
@@ -50,9 +50,9 @@
     end
 
     when_invoked do |options|
-      raise "Not a CA" unless Puppet::SSL::CertificateAuthority.ca?
+      raise _("Not a CA") unless Puppet::SSL::CertificateAuthority.ca?
```

Comment:
I think I might actually make a note that CA stands for certificate authority, and let them decide how to handle it. Might be nice to give it meaning, rather than not translate it.